db/models/Comment.js

```diff
@@ -9,11 +9,11 @@ module.exports = (sequelize, DataTypes) => {
   Comment.associate = (models) => {
     Comment.belongsTo(models.User, {
       foreignKey: 'userId',
-      as: 'user'
+      as: 'author'
```

Comment:
`as: user` should remain and not `as: author` user is to comment while author is to article

Reply:
Thank you for your feedback. Following the specs on the [README](https://github.com/andela/demeter-ah-backend) file, when an article is retrieved, it should come along with the author of the article and the comments should come along with authors who made the comment with a key value of `author`.  So for maintaining uniformity, since the default users are authors on our platform. it is advantageous to the frond-devs to use `author` though-out, just as we have agreed as a team.